setup.py

```diff
@@ -22,7 +22,7 @@
 ]
 
 extras_require = {
-    "format" : ["rfc3987", "strict-rfc3339", "webcolors"],
```

Comment:
I didn't think of checking the other deps... `strict-rfc3339` is GPL as well so we'll need to find an alternative there, `webcolors` is BSD.

Totally reasonable for the backwards compatibility perspective.